### PR TITLE
Export images with webhook icon

### LINKED ISSUE #4127: Export to PNG of a diagram with webhook connector fails
### Describe the bug

In Camunda Modeler 5.20.0, the attached BPMN fails to export to PNG, and an error is printed:

```
This page contains the following errors:error on line 1 at column 723: Unescaped '<' not allowed in attributes values
Below is a rendering of the page up to the first error. [ ERROR ]
```

[insurance.bpmn.txt](https://github.com/camunda/camunda-modeler/files/14239778/insurance.bpmn.txt)

When I remove the event with custom icon, it exports correctly.

### Steps to reproduce

1. Open the attached BPMN
2. Try to export to PNG
3. now this happens

### Expected behavior

It should export.

### Environment

- OS: MacOS
- Camunda Modeler Version: 5.20.0
- Execution Platform: [e.g. Camunda 7, Camunda 8]
- Installed plug-ins: [...]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by barmac**
- fix: export images with webhook icon
- deps: update to `tiny-svg@3.1.3`

Closes #4127

https://github.com/user-attachments/assets/3c851840-c972-4d26-af2f-5499fbb77c8f

### Proposed Changes

This updates tiny-svg and also includes it in the dependencies as we already use it in the source code: https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/dmn/modeler/features/overview/overview-renderer/OverviewRenderer.js#L28

### Checklist

To ensure you provided everything we need to look at your PR:

* [x] __Brief textual description__ of the changes present
* [x] __Visual demo__ attached
* [ ] __Steps to try out__ present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [x] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -67,6 +67,7 @@
     "scroll-tabs": "^1.0.1",
     "semver-compare": "^1.0.0",
     "sourcemapped-stacktrace": "^1.1.11",
+    "tiny-svg": "^3.1.3",
```
Comment (nikku) [inline client/package.json:70]:
```suggestion
    "tiny-svg": "^3.1.3",
```

Should we not just `npm update tiny-svg` to update the dependency (nested)?

Comment (nikku) [reply client/package.json:70]: Feels odd to introduce this here (unless we use it within the modeler).

Comment (nikku) [reply client/package.json:70]: (We do use it, but it is pulled in through `bpmn-js`, so I'd expect to simply bump the nested dependency).

Comment (barmac) [reply client/package.json]: I pulled it because we directly depend on it. I should've commented on that.

**Pull request merged**